SmartDeviceLink/public/SDLTextFieldName.h

```diff
@@ -82,7 +82,7 @@ extern SDLTextFieldName const SDLTextFieldNameStatusBar;
  *
  * @discussion This field is commonly used to show elapsed or remaining time in an audio track or audio capture.
  */
-extern SDLTextFieldName const SDLTextFieldNameMediaClock;
+extern SDLTextFieldName const SDLTextFieldNameMediaClock __deprecated;
```

Comment:
The generated comment for this should include information about when it was deprecated. Please include that information in this comment.